lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -102,6 +104,25 @@ class DetailsRenderer {
     element.appendChild(cardsParent);
     return element;
   }
+
+  /**
+   * @param {!DetailsRenderer.CodeDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderCode(details) {
+    const element = this._dom.createElement('div', 'lh-details');
+
+    const pre = this._dom.createElement('pre', 'lh-code');
+    pre.textContent = details.text;
+
+    if (details.header) {
```

Comment:
I was using `label` but decided to keep it consistent with the rest of the details renderers. And it's more or less a "header" for the code snippet.